app/serializers/group_user_assigned_serializer.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class GroupUserAssignedSerializer < BasicUserSerializer
+  include UserPrimaryGroupMixin
+
+  attributes :name,
+             :title,
+             :last_posted_at,
+             :last_seen_at,
+             :added_at,
+             :assignments_count,
+             :timezone
```

Comment:
Do we need all these attributes? `name` is already included in BasicUserSerializer. And I don't think we're actually using title/last_posted_at/last_seen_at/added_at/timezone for the assign UI?